Sources/tart/Commands/Get.swift

```diff
@@ -0,0 +1,31 @@
+import ArgumentParser
+import Foundation
+
+struct Get: AsyncParsableCommand {
+  static var configuration = CommandConfiguration(commandName: "get", abstract: "Get a VM's configuration")
+
+  @Argument(help: "VM name")
+  var name: String
+
+  func run() async throws {
+    do {
+      let vmDir = try VMStorageLocal().open(name)
+      let vmConfig = try VMConfig(fromURL: vmDir.configURL)
+      let diskSize = try vmDir.sizeBytes() / 1024 / 1024 / 1000
```

Comment:
That calculation looks a bit weird to me, why both 1024 and 1000? 🤔 

https://github.com/cirruslabs/tart/blob/555715588d4794fa0875d8dc3998bda0099bd77d/Sources/tart/VMDirectory.swift#L85

What do you think about showing the output in GiB's instead?

```suggestion
      let diskSize = try vmDir.sizeBytes() / 1000 / 1000 / 1000
```

Reply:
In retrospect that was a weird way to show GB. I like the idea of GiB instead, it's more accurate